src/evaluators/SwitchStatement.js

```diff
@@ -90,32 +56,32 @@ function AbstractCaseBlockEvaluation(
       for (let i = 0; i < c.consequent.length && !(result instanceof Completion); ++i) {
```

Comment:
!(result instanceof Completion) is always true here.

Reply:
Ah yes...leftover from moving out the breaks. I'll kill it off, thanks.